zeppelin-server/src/main/java/org/apache/zeppelin/server/ZeppelinServer.java

```diff
@@ -298,6 +301,27 @@ public void contextDestroyed(ServletContextEvent servletContextEvent) {}
     }
   }
 
+  private static void validateShiroIni() throws Exception {
+    if (conf.getShiroPath().length() > 0) {
+      Ini ini = new Ini();
+      ini.loadFromPath(conf.getShiroPath());
+
+      Ini.Section usersSection = ini.get("users");
+      String activeDirectoryRealm = ini.getSectionProperty("main", "activeDirectoryRealm");
```

Comment:
I see two problems in this approach
 - This is just a name user can choose it to be anything, for example `activeDirectoryRealm` user can choose to write this as just `adRealm`
 - What happens if in the future community adds a new realm?

Reply:
> This is just a name user can choose it to be anything, for example activeDirectoryRealm user can choose to write this as just adRealm

I thought these properties names are constant means users can not change it.